modules/admin_manual/pages/installation/linux_installation.adoc

```diff
@@ -1,14 +1,85 @@
 = Linux Package Manager Installation
+:apt-mark-hold-url: https://manpages.debian.org/stretch/apt/apt-mark.8.en.html#PREVENT_CHANGES_FOR_A_PACKAGE
 
 NOTE: Package managers should only be used for single-server setups. For production environments, we recommend installing from
 https://owncloud.org/download/#owncloud-server-tar-ball[the tar archive].
 
 [[available-packages]]
 == Available Packages
 
-The recommended package to use is `owncloud-files`. It only installs
-ownCloud, and does not install Apache, a database, or any of the
-required PHP dependencies.
+The recommended package to use is `owncloud-files`. 
+It only installs ownCloud, and does not install Apache, a database, or any of the required PHP dependencies.
+
+== Avoid Automatic Upgrades
+
+If you are installing ownCloud using one of the various Linux package managers, we *strongly* recommend that you avoid automatically updating the `owncloud-files` package, when running a system update or upgrade and when upgrading other packages.
+That way, there are no surprise changes (whether positive or negative) to your ownCloud installation.
+
+Here are the ways to do so for xref:apt[APT], xref:yum[Yum], and xref:zypper[Zypper].
+
+=== APT
+
+If you are using APT, use {apt-mark-hold-url}[apt-mark hold] to  mark the `owncloud-files` package as held.
+Here’s an example of how to do so:
+
+[source,console]
+....
+apt-mark hold owncloud-files
+....
+
+To see if owncloud-files has already been held, use the `showhold` command, as in the following example.
+If it’s printed out to the console, then it’s being held.
+
+[source,console]
+....
+apt-mark showhold owncloud-files
+....
+
+To unset `owncloud-files` as held back, use the `unhold` command, as in the example below.
+
+[source,console]
+....
+apt-mark unhold owncloud-files
+....
+
+=== Yum
+
+If you are using Yum, use the `-x` or `--exclude` options.
+These two options exclude the package from update, installation, and removal.
+Here’s an example when running an update:
+
+[source,console]
+....
+yum update -x owncloud-files
```

Comment:
Sorry to pick on this - but this is only temporarily and the admin needs to remember to do this everytime.

The other options would be:
- edit `yum.conf` and add `exclude=owncloud-files`
- use `yum-versionlock` http://man7.org/linux/man-pages/man1/yum-versionlock.1.html

Reply:
Don't worry about being nitpicky. It's what makes projects as professional as possible. Your knowledge of Yum's better than mine. I stick to APT, mainly. And now I've learned more.

Reply:
Updated.